core/templates/dev/head/domain/exploration/SolutionObjectFactory.js

```diff
@@ -88,10 +88,11 @@ oppia.factory('SolutionObjectFactory', [
 
     Solution.prototype.getOppiaResponseHtml = function(interaction) {
       return (
-        (this.answerIsExclusive ? 'The only' : 'One') + ' answer is:<br>' +
+        '<label>' + (this.answerIsExclusive ? 'The only' : 'One') +
+        ' answer is: &nbsp;</label>' +
```

Comment:
Ah no -- can we find a way to do this such that all the HTML is in the template files? Doing manual construction of HTML in JS is a minefield for security issues. It's OK to declare new JS functions if needed (to get the info needed to display the component parts of the displayed output).

Reply:
Ok, how about, getting short answer and long answer separately and displaying under labels in HTML files?

Reply:
Yep, that sounds fine! Just ensure that there's no direct HTML construction in the JS.

Reply:
Ok, will do that.

Reply:
Done, also added a file in exploration_editor.html as solution was not loading in preview tab of editor